spec/features/omniauth_spec.rb

```diff
@@ -49,6 +49,17 @@
       expect(flash).to eq('Could not authenticate you from Google because "Invalid credentials".')
     end
 
+    scenario 'should not signup when email is not there in auth hash' do
```

Comment:
why a feature test instead a controller test? :thinking:

Reply:
@Ana06 I guess there is already a file of openid in features so I thought it would be good to write the tests there only. Also I noticed that all flash message related tests were written in feature folder. If required want I can make it in controllers too. Please correct me if I'm wrong!

Reply:
In the feature test I would be testing that no new Openid record is created.
I would add a controller test to test the redirection.

Reply:
@differentreality Got it!!I'll make the changes asap

Reply:
Please do add/amend the tests, so that we can merge it! Thanks @ViditChitkara